src/style-spec/reference/v8.json

```diff
@@ -2959,7 +2959,7 @@
         }
       },
       "image": {
-        "doc": "Returns an `image` type for use in `icon-image`, `*-pattern` entries and as a section in the `format` expression. If set, the `image` argument will check that the requested image exists in the style and will return either the resolved image name or `null`, depending on whether or not the image is currently in the style. This validation process is synchronous and requires the image to have been added to the style before requesting it in the `image` argument.",
+        "doc": "Returns a [`ResolvedImage`](../types/#resolvedimage) for use in [`icon-image`](../layers/#layout-symbol-icon-image), `*-pattern` entries and as a section in the [`'format'`](#types-format) expression. A [`'coalesce'`](#coalesce) expression containing `image` expressions will evaluate to the first listed image that is currently in the style. This validation process is synchronous and requires the image to have been added to the style before requesting it in the `'image'` argument.",
```

Comment:
I _think_ that these links should be formatted as `/mapbox-gl-js/style-spec/etc` rather than using the `../etc` notation 🤔  is that right @katydecorah?

Reply:
Yes! The full relative path is preferred. Thank you, Heather!

Reply:
Good catch @HeyStenson and @katydecorah!